model/team_member.go

```diff
@@ -69,10 +69,12 @@ func (o *TeamUnread) ToJson() string {
 	return string(b)
 }
 
-func TeamMemberFromJson(data io.Reader) *TeamMember {
+func TeamMemberFromJson(data io.Reader) (*TeamMember, error) {
```

Comment:
Do we consider this a *breaking change* that should only go in a major release?
I've tried to do the *right* thing here and avoid ignoring the error but I was left with this doubt.

Reply:
@jespino 

Do you think it makes sense to add a `TeamMemberFromJSON` (notice the uppercase at the end) version that handles the error and start the deprecation of `TeamMemberFromJson`? Or were you suggesting to just add an internal (unexported) method?

Reply:
Alright, keeping it simple then.